src/main/java/LacksDataBaseProject/AddAndRemoveSupplier.java

```diff
@@ -0,0 +1,35 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveSupplier {
+    static Scanner input = new Scanner(System.in);
+
+    public static void addSupplier(Supplier supplier) {
+        System.out.println("Supplier name: ");
+        String supplierName = input.next();
+        System.out.println("Select Purchaser from List, give Skype ID: ");
+        for (User user : User.listOfUsers) {
+            if (user.getRole().equals(Role.PURCHASER)) {
+                System.out.println("\t" + user);
+                String skypeID = input.next();
```

Comment:
Tutaj jesli wybieramy Purchaser'a z listy to chyba powinien mu się skypeID z automatu nadawać - po co to wpisywać jak to jest już w bazie.